prov/verbs/src/verbs_domain.c

```diff
@@ -308,21 +308,23 @@ static struct fi_ops_domain vrb_dgram_domain_ops = {
 	.query_collective = fi_no_query_collective,
 };
 
-static struct fi_info *
-vrb_get_verbs_info(const char *domain_name)
+static int
+vrb_check_verbs_info(struct fid_fabric *fabric, struct fi_info *info)
 {
 	const struct fi_info *fi;
+	int ret = -FI_ENODATA;
 
 	ofi_mutex_lock(&vrb_info_mutex);
 	for (fi = vrb_util_prov.info; fi; fi = fi->next) {
-		if (!strcmp(fi->domain_attr->name, domain_name)) {
-			ofi_mutex_unlock(&vrb_info_mutex);
-			return fi_dupinfo(fi);
+		if (!strcmp(fi->domain_attr->name, info->domain_attr->name)) {
+			ret = ofi_check_domain_attr(&vrb_prov, fabric->api_version,
+					fi->domain_attr, info);
```

Comment:
please align the arguments to keep it consistent with other places in the same file.

Reply:
Thanks for the quick review! This is fixed now.